src/index.html

```diff
@@ -1,13 +1,628 @@
+<!-- #region head -->
 <!DOCTYPE html>
-<html lang="en">
+<html lang="en" class="page">
   <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Dia</title>
+    <title>Air</title>
+    <link
+      rel="icon"
+      type="icon"
+      href="./images/icon/favicon.ico"
+    >
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Open+Sans:wght@400;600&family=Poppins:wght@400;600;700&display=swap" rel="stylesheet">
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
-  <body>
-    <h1>Dia</h1>
+  <!-- #endregion -->
+  <body class="page__body">
+    <header class="header">
+      <div class="header__top">
+        <div class="top-bar">
+          <div class="top-bar__wrapper">
+            <a href="#" class="top-bar__logo-link">
+              <img
+                src="images/logo/air-logo.svg"
+                alt="logo"
+                class="top-bar__logo"
+                >
+            </a>
+
+            <a href="#menu" class="icon icon--menu"></a>
+          </div>
+          <nav class="nav top-bar__nav">
+            <ul class="nav__list nav__list--top-bar">
+              <li class="nav__item">
+                <a href="#about-us" class="nav__link">About us</a>
+              </li>
+              <li class="nav__item">
+                <a href="#services" class="nav__link">Services</a>
+              </li>
+              <li class="nav__item">
+                <a href="#testimonials" class="nav__link">Testimonials</a>
+              </li>
+              <li class="nav__item">
+                <a href="#contact-us" class="nav__link">Contact Us</a>
+              </li>
+            </ul>
+          </nav>
+          <a href="#contact-us" class="top-bar__hire-us">Hire us</a>
+        </div>
+      </div>
+
+      <div class="header__bottom">
+        <div class="header__section">
+          <section class="strategic-agency">
+            <h1 class="strategic-agency__title">Strategic Agency</h1>
+            <p class="strategic-agency__description">
+              We believe in the power of bold ideas that can solve business challenges.
+            </p>
+            <a href="#" class="main-button strategic-agency--button">Learn more</a>
+          </section>
+        </div>
+
+        <div class="header__slider">
+          <div class="slider header__slider">
+            <ul class="slider__list">
+              <li id="slider-1" class="slide slide--1">
+                <div class="slide__button-wrapper">
+                  <a class="button button--prev" href="#slider-3">
+                    <img
+                      class="button__arow button__arow--prev"
+                      src="./images/slider/mdi_chevron_left.svg" alt="arrow-left"
+                      >
```

Comment:
Need to fix it everywhere
```suggestion
                    <img
                      class="button__arow button__arow--prev"
                      src="./images/slider/mdi_chevron_left.svg" alt="arrow-left"
                    >
```